js/script_yt.js

```diff
@@ -0,0 +1,36 @@
+console.log("hello1");
+console.log("hello2");
+
+var API_KEY="AIzaSyC5kvxuhyuLX5No7WCurPoeVimBhtTRTok";
```

Comment:
Same as above. Please wrap the code in a function to avoid variable conflicts and avoid global variables